e2e/celestia-app/config.toml

```diff
@@ -381,13 +381,13 @@ timeout_propose = "10s"
 # How much timeout_propose increases with each round
 timeout_propose_delta = "500ms"
 # How long we wait after receiving +2/3 prevotes for “anything” (ie. not a single block or nil)
-timeout_prevote = "1s"
+timeout_prevote = "100ms"
 # How much the timeout_prevote increases with each round
-timeout_prevote_delta = "500ms"
+timeout_prevote_delta = "1ms"
 # How long we wait after receiving +2/3 precommits for “anything” (ie. not a single block or nil)
-timeout_precommit = "1s"
+timeout_precommit = "100ms"
 # How much the timeout_precommit increases with each round
-timeout_precommit_delta = "500ms"
+timeout_precommit_delta = "1ms"
```

Comment:
not sure that changing some of these times such as the precommit or the deltas will do that much, but increasing the prevote time definitely makes sense

we might need to build a special version of the app that doesn't have overrides :thinking:

Reply:
ahh I see, makes sense then.